server/src/entity/falling_block.rs

```diff
@@ -0,0 +1,19 @@
+use feather_blocks::Block;
+use specs::{Component, VecStorage};
+
+/// Component for falling block entities.
+pub struct FallingBlockComponent {
+    pub block: Block,
+}
+
+impl Default for FallingBlockComponent {
+    fn default() -> Self {
+        FallingBlockComponent {
+            block: Block::Stone,
+        }
+    }
+}
+
+impl Component for FallingBlockComponent {
+    type Storage = VecStorage<Self>;
```

Comment:
Probably use DenseVecStorage rather than VecStorage here, since most entities won't be falling blocks. VecStorage would waste memory in this case.

Reply:
Should `ItemComponent` be changed too, then?

Reply:
Yes, I guess that's probably true. Not sure what I was thinking when I chose that storage, sorry for misleading.